docs/advanced/config_system.md

```diff
@@ -9,6 +9,8 @@ The mbed configuration system customizes the compile time configuration of vario
 
 The configuration system gathers and interprets all the configurations defined in the source tree. The output of the configuration system is a list of macros that are automatically defined when compiling the code.
 
+<span class="notes">**Note:** In prior releases, the configuration system provided a method for adding custom targets. The mbed OS tools look for custom targets in a file named `custom_targets.json` in the root of a project, and treat them the same as [mbed targets](mbed_targets.md).</span>
```

Comment:
Significantly better. Please delete the comma after "project". Also, should we clarify "them" refers to "custom targets", or do you think it's OK?